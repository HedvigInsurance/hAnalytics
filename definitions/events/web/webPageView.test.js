const validateEvent = require("commons/validateEvent.js");

test("validate", () => {
  const event = validateEvent(__filename, {
    href: "http://www.hedvig.com/page",
    pathname: "/page",
    hostname: "www.hedvig.com",
  });
  expect(event).toMatchSnapshot();
});
