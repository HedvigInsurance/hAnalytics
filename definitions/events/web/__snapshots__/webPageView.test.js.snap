// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "web_page_view",
  "variables": Object {
    "hostname": "www.hedvig.com",
    "href": "http://www.hedvig.com/page",
    "pathname": "/page",
  },
}
`;
